packages/adp-tooling/templates/project/package.json

```diff
@@ -10,17 +10,19 @@
     ],
     "dependencies": {},
     "devDependencies": {
-        <%if (locals.options?.fioriTools) {%>"@sap/ux-ui5-tooling": "^1.11.4"<%} else {%>"@sap-ux/backend-proxy-middleware": "^0.7.5",
+        <%if (locals.options?.fioriTools) {%>"@sap/ux-ui5-tooling": "1"<%} else {%>"@sap-ux/backend-proxy-middleware": "^0.7.5",
         "@sap-ux/preview-middleware": "^0.11.1",
         "@sap-ux/ui5-proxy-middleware": "^1.3.0",
         "@sap-ux/deploy-tooling": "^0.11.7"<%}%>,
-        "@ui5/cli": "^3.9.1"
+        "@ui5/task-adaptation": "^1.3.0",
+        "@ui5/cli": "^3.9.2"
     },
     "scripts": {
         "build": "ui5 build --exclude-task generateFlexChangesBundle generateComponentPreload --clean-dest",
```

Comment:
Could you please add `minify` to the excluded task? Otherwise, `.map` files will be generated that are rejected by LREP on your ABAP versions

Reply:
Done